### DISCUSSION TITLE
"Execution model" isn't the word. What's the word for the map that shows what methods are called and in what order?

### DISCUSSION THREAD

**Discussion by axtens** [unoplatform/uno#14589] [Q&A]
My difficulty is that in the IDE we're developing, after everything has started we're left with a rotating ring cursor. No matter where I put a call to ` .Focus(FocusState.Keyboard)`, the focus hasn't moved. I actually have to click on the RichEditBox for the visual clue that is the caret to appear. How do I set the focus. It's different when I create another tab: we're no longer straight out of "launch mode".

(Is Uno/WinUI/XAML/MVVM supposed to be making things simpler? I'm not convinced.)

**Comment by jeromelaban** [comment]
Thanks for the question. 

This is a bit difficult to help you, there is not enough information. If you have a repro, which platform and version you're using, it will help greatly.

**Reply by axtens** [reply]
Platform: Windows 11, Visual Studio 2022 Community
Version: Uno.Core 4.0.1 

There are so many different packages. I'm not at all sure which one defines the version. Certainly we haven't made the jump to v5 yet. 

As for the "repro" ... hmmm ... that might mean rewriting the entire app so that I can set up all the weird interactions that the event model entails. Or I could give you access to the maniac's breakfast that is the current project.

**Reply by jeromelaban** [reply]
When we ask for a version, we generally mean Uno.UI or Uno.WinUI. As for having access to your sources, we generally don't work directly in app's sources, but you can contact us at info@platform.uno as we offer professional services around this kind of cases.
